index.js

```diff
@@ -38,15 +41,15 @@ class ResembleHelper extends Helper {
                 });
             }
 
-            if (options.tolerance !== undefined)
+            if (options.tolerance !== undefined && this.config.consoleOutput)
             {
                 console.log("Tolerance Level Provided " + options.tolerance);
                 var tolerance = options.tolerance;
```

Comment:
In case the user doesn't want to see the output on Console, this will also not allow initializing the tolerance, making `tolerance` undefined.

Reply:
Updated